mitodl/mitxpro
# Replace Icomoon & font-awesome with Google Material Fonts

#### Pre-Flight checklist



#### What are the relevant tickets?
https://github.com/mitodl/mitxpro/issues/2065

#### What's this PR do?
Replaces the usage of Icomoon & font-awesome with Google Material Icons.

#### How should this be manually tested?
Need to verify the icons and their usage is intact.
Need to verify that there is not a lot of difference between the look & feel of the Icons used through Google Material Font


#### Any background context you want to provide?
In past we were using icons from 3 different sources Google Material, Icomoon & Font-Awesome. Now, While we are in the process of improving the performance of the website so we decided to use the icons just from Google Material Fonts which will provide use some performance improvement since we'll have to load just Google Font reducing the page load time taken by Icomoon & Font-Awesome.


#### Screenshots (if appropriate)

1) Notification Cancel Icon
**Old**
<img width="1440" alt="old_notification_cancel" src="https://user-images.githubusercontent.com/34372316/106276295-33774d00-6259-11eb-9550-d90aefd76be3.png">

**New** 
<img width="1439" alt="new_notification_cancel" src="https://user-images.githubusercontent.com/34372316/106276369-4853e080-6259-11eb-9554-f335d4703f79.png">

2) Course Carousel
**Old** 
<img width="1422" alt="old_course_carousel_icons" src="https://user-images.githubusercontent.com/34372316/106276273-2d816c00-6259-11eb-8e3a-202cc3e2b17a.png">

**New**
<img width="1439" alt="new_course_carousel_icons" src="https://user-images.githubusercontent.com/34372316/106276354-44c05980-6259-11eb-8064-a5c929c25ff5.png">

3) Testimonials Carousel
**Old** 
<img width="1415" alt="old_testimonial_carousel_icons" src="https://user-images.githubusercontent.com/34372316/106276280-2fe3c600-6259-11eb-8752-65de64233d79.png">

**New**
<img width="1422" alt="new_testimonial_carousel_icons" src="https://user-images.githubusercontent.com/34372316/106276361-468a1d00-6259-11eb-8325-cbc06b333e09.png">

4) Catalog Details
**Old**
<img width="955" alt="old_catalog_detail_icon" src="https://user-images.githubusercontent.com/34372316/106276269-2c503f00-6259-11eb-9332-462765c54715.png">

**New**
<img width="899" alt="new_catalog_detail_icons" src="https://user-images.githubusercontent.com/34372316/106276349-42f69600-6259-11eb-85cf-172003730ef7.png">
 
5) Testimonials Dialog Cancel
**Old** 
<img width="579" alt="old_testimonial_cancel" src="https://user-images.githubusercontent.com/34372316/106276287-3114f300-6259-11eb-85b0-f73bf6ab0950.png">

**New**
<img width="528" alt="new_testimonial_cancel" src="https://user-images.githubusercontent.com/34372316/106276365-47bb4a00-6259-11eb-8643-da71229912b9.png">

6) Back to top
**Old**
<img width="967" alt="old_back_to_top_button" src="https://user-images.githubusercontent.com/34372316/106276257-278b8b00-6259-11eb-898c-35f0b7c8a0f6.png">

**New**
<img width="934" alt="new_back_to_top_icon" src="https://user-images.githubusercontent.com/34372316/106276329-3eca7880-6259-11eb-9cc3-7e12e079a269.png">


**Mobile Screenshots**
1) Notification's Cancel Button
**Old**
<img width="361" alt="old_notification_cancel_mobile" src="https://user-images.githubusercontent.com/34372316/106280105-468d1b80-625f-11eb-85f0-1f29957c8e89.png">

**New**
<img width="360" alt="new_notification_cancel_mobile" src="https://user-images.githubusercontent.com/34372316/106280055-35440f00-625f-11eb-974e-b9fce8f58100.png">

2) Course Carousel
**Old**
<img width="361" alt="old_course_carousel_mobile" src="https://user-images.githubusercontent.com/34372316/106280096-442ac180-625f-11eb-911d-2c551796d7ce.png">

**New**
<img width="361" alt="new_course_carousel_mobile" src="https://user-images.githubusercontent.com/34372316/106280050-337a4b80-625f-11eb-935c-7afdb6dfd4d6.png">

3) Testimonials Carousel
**Old**
<img width="361" alt="old_testimonials_mobile" src="https://user-images.githubusercontent.com/34372316/106280091-412fd100-625f-11eb-9e49-431e85550ab5.png">

**New**
<img width="363" alt="new_testimonials_carousel_mobile" src="https://user-images.githubusercontent.com/34372316/106280052-3412e200-625f-11eb-8b29-1cdc132ea2d9.png">

4) Catalog Details
**Old**
<img width="360" alt="old_catalog_detail_mobile" src="https://user-images.githubusercontent.com/34372316/106280107-4725b200-625f-11eb-91b7-13613b8ca9bd.png">

**New**
<img width="362" alt="new_catalog_detail_mobile" src="https://user-images.githubusercontent.com/34372316/106280046-307f5b00-625f-11eb-9c58-e85a651b1cbf.png">

5) Testimonial Dialog Cancel
**Old**
<img width="357" alt="old_testimonial_cancel_mobile" src="https://user-images.githubusercontent.com/34372316/106280082-3e34e080-625f-11eb-8625-848807132fde.png">

**New**
<img width="367" alt="new_testimonial_cancel_mobile" src="https://user-images.githubusercontent.com/34372316/106280054-34ab7880-625f-11eb-94a3-6bdd148e819b.png">

6) Back to Top Button
**Old**
<img width="363" alt="old_back_to_top_mobile" src="https://user-images.githubusercontent.com/34372316/106280112-4856df00-625f-11eb-9ee2-a56835f5552b.png">

**New**
<img width="360" alt="new_back_to_top_mobile" src="https://user-images.githubusercontent.com/34372316/106280029-2c533d80-625f-11eb-9061-cc420621ec68.png">
